src/Microsoft.Azure.ServiceBus/Message.cs

```diff
@@ -44,10 +52,10 @@ public Message(byte[] array)
         /// <remarks>
         /// The easiest way to create a new message from a string is the following:
         /// <code>
-        /// message.Body = System.Text.Encoding.UTF8.GetBytes("Message1");
+        /// message.Body = new ArrySegment&lt;byte&gt;(System.Text.Encoding.UTF8.GetBytes("Message1"));
         /// </code>
         /// </remarks>
-        public byte[] Body { get; set; }
+        public ArraySegment<byte> Body { get; set; }
```

Comment:
> so let's move byte[] ctor?

Not sure I follow. Did you mean "remove"?

> Yes, this is a breaking change

Well, this is a pre-release 🙂 


`ArraySegment` - not arguing. Probably internally makes a lot of sense. Externally perhaps should be `byte[]` as the first/default option.